packages/client/hmi-client/src/services/distribution.ts

```diff
@@ -1,15 +1,196 @@
+	// [DistributionType.Laplace1]: ['location', 'scale'],
+	// [DistributionType.Laplace2]: ['location', 'precision'],
+	// [DistributionType.ParetoTypeI1]: ['scale', 'shape'],
+	// [DistributionType.Poisson1]: ['rate'],
+	// [DistributionType.StudentT1]: ['degreesOfFreedom'],
+	// [DistributionType.StudentT2]: ['mean', 'scale', 'degreesOfFreedom'],
+	// [DistributionType.StudentT3]: ['degreesOfFreedom', 'location', 'scale'],
+	// [DistributionType.Weibull1]: ['scale', 'shape'],
+	// [DistributionType.Weibull2]: ['scale', 'shape']
+};
+
 export const DistributionTypeLabel: { [key in DistributionType]: string } = {
 	[DistributionType.Constant]: 'Constant',
 	[DistributionType.Uniform]: 'Uniform'
+
+	// Disabled until the backend is updated
+	// [DistributionType.StandardNormal1]: 'Standard',
+	// [DistributionType.Normal1]: 'Normal 1',
+	// [DistributionType.Normal2]: 'Normal 2',
+	// [DistributionType.Normal3]: 'Normal 3',
+	// [DistributionType.LogNormal1]: 'LogNormal 1',
+	// [DistributionType.LogNormal2]: 'LogNormal 2',
+	// [DistributionType.Bernoulli1]: 'Bernoulli 1',
+	// [DistributionType.Bernoulli2]: 'Bernoulli 2',
+	// [DistributionType.Beta1]: 'Beta 1',
+	// [DistributionType.BetaBinomial1]: 'BetaBinomial 1',
+	// [DistributionType.Binomial1]: 'Binomial 1',
+	// [DistributionType.Binomial2]: 'Binomial 2',
+	// [DistributionType.Cauchy1]: 'Cauchy 1',
+	// [DistributionType.ChiSquared1]: 'ChiSquared 1',
+	// [DistributionType.Dirichlet1]: 'Dirichlet ',
+	// [DistributionType.Exponential1]: 'Exponential 1',
+	// [DistributionType.Exponential2]: 'Exponential 2',
+	// [DistributionType.Gamma1]: 'Gamma 1',
+	// [DistributionType.Gamma2]: 'Gamma 2',
+	// [DistributionType.InverseGamma1]: 'InverseGamma 1',
+	// [DistributionType.Gumbel1]: 'Gumbel 1',
+	// [DistributionType.Laplace1]: 'Laplace 1',
+	// [DistributionType.Laplace2]: 'Laplace 2',
+	// [DistributionType.ParetoTypeI1]: 'ParetoType I 1',
+	// [DistributionType.Poisson1]: 'Poisson 1',
+	// [DistributionType.StudentT1]: 'StudentT 1',
+	// [DistributionType.StudentT2]: 'StudentT 2',
+	// [DistributionType.StudentT3]: 'StudentT 3',
+	// [DistributionType.Weibull1]: 'Weibull 1',
+	// [DistributionType.Weibull2]: 'Weibull2'
 };
 
 export const distributionTypeOptions = (): { name: string; value: string }[] =>
 	Object.values(DistributionType).map((dist) => ({
 		name: DistributionTypeLabel[dist],
 		value: dist
 	}));
+
+export const distributionDescription = {
+	[DistributionType.Constant]: 'Constants description',
+	[DistributionType.Uniform]: 'Generates uniformly distributed random samples from the half-open interval.'
+
+	// Disabled until the backend is updated
+	// [DistributionType.StandardNormal1]: 'StandardNormal1 description',
+	// [DistributionType.Normal1]: 'Normal1 description'
+};
+
+const externaDoclLink = 'https://pytorch.org/docs/stable/distributions.html';
+
+export const externaDoclLinks = {
+	[DistributionType.Uniform]: `${externaDoclLink}#uniform`
+};
```

Comment:
Food for thoughts:

When we chatted I didn't realize there would be so many parallel data structures being used, I think we are okay for this PR, but we should consider consolidating in the next-round so the substructures of each distribution objects are closer together. 

The issue with parallel data structure is that you need to actively remember to change in all the places, and it can be a bit more error prone.

We can chat again if it helps, offhand maybe something like this can suffice. 

```
const distributions = {
  [DistributionType.uniform] : {
    label: ...,
    parameters: [
      {
        label: "Minimum",
        name: "mininum"
      },
      {
        label: "Maximum",
        name: "maximum"
      }
    ]
  }
}

```